### PR TITLE
Ariellaw donations test

### PR DESCRIPTION
**Pull request by Ariellaw**
I wrote tests for the donations page - Eyal already reviewed it.
I plan to enhance it further.

### CODE REVIEW

**Review by MaorUziel — COMMENTED** [review]
It's looking good :) 
So I see this test check if the text and elements appear in the donations pages as excepted. When Eyal talked about donation test I thought he meant to make a donation and see if the donation success(cardcom accepted the donation and approve it). maybe that what you mean enhance the test further?
I really think the test you made is the part that was missing in my tests. I only check if the sign ups transferred to the places they should, but sometimes test can fail because someone edited the page with the form that the bot checking. Because the page changed selenium can raise exception or worse, tell us it filled and send the form while he didn't. So checking the page first will make debugging failures easier.

File: DonationTest.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    def setUp(self):
+        random_num = random.randint(1, 11)
+        print(random_num)
+        if random_num <= 5:
+            self.driver = webdriver.Firefox()
+        else:
+            self.driver = webdriver.Chrome()
+
+    def test_donation_form(self):
+        pages = [Page("https://animals-now.org/donate/?group=test", "תרומה חודשית",  ["50","120", "200", "400"], "₪" ),
```
[the remark this answers is not in the document]
Comment (Ariellaw) [reply DonationTest.py:27]: Thanks, I made this change. :)

**Comment by Ariellaw** [comment]
Thanks for the feedback.

I am writing a simple test (just checking the texts) for the 2nd page at the moment. 

I will test a full flow end to end (including email) in a separate PR.

**Comment by Ariellaw** [comment]
I am done and ready to merge 😺

**Review by pinkasey — APPROVED** [review]
@MaorUziel 
As far as I'm concerned - this can be merged.
Only you can merge this, since it's still your repository.

Once this is merged, I'll create a jenkins job for it.

**Comment by pinkasey** [comment]
wow that was fast...

**Pull request merged**